### ISSUE TITLE
Issue with GetBatches function

### ISSUE THREAD

**Issue by SnehaChou** [malllabiisc/WordGCN#29]
I am trying to replicate your code with the same dataset which you used for training. My code stops running once it enters run_epoch function in syngcn.py code.
The issue seems to appear in self.getBatches(shuffle), however I ran make command and BatchGen.so is created. So not really sure why my code stops running without any error. 
<img width="1038" alt="Screen Shot 2020-03-13 at 10 27 40 PM" src="https://user-images.githubusercontent.com/36100392/76673421-39017100-657b-11ea-9bc3-cc0dc9f0ccec.png">

**Comment by cairouchong** [comment]
I think you can try to check batch_generator.cpp. The path of the data.txt in reset() is correct or not. If you  change it, please run make command again. @SnehaChou

**Comment by SnehaChou** [comment]
Thanks @cairouchong. I checked the path of reset function in batch_generator.cpp and the path was correct. However when I commented the line "self.lib.reset()" in syngcn.py, the code starts running fine. Not sure whether it is the right way to do it. 

Also, even after 25 epochs my training loss is nan. Is it normal or because of some other issue?
<img width="743" alt="Screen Shot 2020-03-14 at 6 17 48 PM" src="https://user-images.githubusercontent.com/36100392/76691349-63593a00-6620-11ea-97ed-6575df9f8cf9.png">

**Comment by svjan5** [comment]
Hi @SnehaChou,
Commenting `self.lib.reset()` should be not be the solution. Also, you should not get `Nan` as the loss, it seems like something is not working correctly. Please verify that all the dependencies are met and the batches are getting generated properly. You can try using `pdb.set_trace()` in batch generation function to see whether you are getting something from cpp file or not.

**Comment by SnehaChou** [comment]
Okay @svjan5. I will look into it. Thanks
